### PR TITLE
Creating map points for users. related card 112

### PR DESCRIPTION
**Pull request by thirdeyeclub**
As a user when an address is added on their profile page it geolocates their address, adds lat/long to their account, and puts them on a map

### CODE REVIEW

File: src/app/page/UserProfile/UserProfile.js [hunk 1]
```diff
@@ -75,9 +76,39 @@ const UserProfile = ({ history }) => {
...
+      !profile.address
+        ? console.log(null)
+        : await addressLookUp(profile.address).then((data) =>
+            firebase.updateProfile({
+              ...profile,
+              addressMapPoint: { latitude: data.lat, longitude: data.long },
+            })
+          );
+    } catch (error) {
+      if (error.response) {
```
Comment (qdozaq) [inline src/app/page/UserProfile/UserProfile.js:91]: Does there need to be any fallback action if we reach this catch? Like telling the user something?

Lets also clean up these console.logs

Comment (azmainamin) [reply src/app/page/UserProfile/UserProfile.js:91]: I would use the ErrorSnackbar here.

File: src/app/page/UserProfile/UserProfile.js [hunk 2]
```diff
@@ -75,9 +76,39 @@ const UserProfile = ({ history }) => {
...
     setView("view");
   }
-  function saveProfileAction(e) {
+  async function saveProfileAction(e) {
     e.preventDefault();
-    firebase.updateProfile(profile);
+    try {
+      !profile.address
+        ? console.log(null)
+        : await addressLookUp(profile.address).then((data) =>
```
Comment (azmainamin) [inline src/app/page/UserProfile/UserProfile.js:84]: wait...why do we have a await and also a .then()...

Comment (thirdeyeclub) [reply src/app/page/UserProfile/UserProfile.js:84]: I'll remove the async / await

**Pull request merged**